src/pages/[platform]/start/getting-started/data-model/index.mdx

```diff
@@ -89,6 +94,23 @@ Choose the following values for each prompt:
 ? Enter the file name for the generated code src/API.ts
 ```
 
+</Block>
+<Block name="JavaScript">
+
+```console
+✔ Are you sure you want to continue? (Y/n) · yes
+...
+? Do you want to generate code for your newly created GraphQL API Yes
+? Choose the code generation language target javascript
+? Enter the file name pattern of graphql queries, mutations and subscriptions src/graphql/**/*.js
+? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions Yes
+? Enter maximum statement depth [increase from default if your schema is deeply nested] 2
+? Enter the file name for the generated code src/API.js
+```
```

Comment:
It seems like adding `>` or something to indicate a customer response would be helpful.